### PR TITLE
Enable the write_lines_files test.

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: src/main/resources/standardTestCases/write_lines_files/write_lines_files.wdl [hunk 1]
```diff
@@ -15,7 +15,10 @@ task switcho_reverso {
   Array[File] files
 
   command {
-    for f in `tac ${write_lines(files)}`
+    # Do this with F to make sure we get a trailing newline for 'tac':
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/write_lines_files/write_lines_files.wdl:18]: TIL `tac`

❌ ⭕️ ❌ 
⭕️ ❌ ⭕️ 
⭕️ ❌ ⭕️

**Comment by mcovarr** [comment]
:+1: for the command block testing.  Should there be a test for `write_lines` in declarations?

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/183/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Review by mcovarr — APPROVED** [review]

**Comment by cjllanwarne** [comment]
I'm holding off on the "in declarations" tests for now, until we decide what `write_lines` should actually be doing in those situations.

**Comment by Horneth** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/183/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**